packages/card/CtaCard.jsx

```diff
@@ -95,7 +96,7 @@ export default function CtaCard({
           >
             {isRealExternalLink ? (
               <ExternalArrowUnstyled
-                title="link is external"
```

Comment:
Previously I added a title attribute to the svg icon, but that was bad of me. When adding title inside SVG it actually isn't an attribute but a `<title>` tag. So I'm using the approach below instead.